internal/apidiff/apidiff.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const rootMod = "cloud.google.com/go"
+
+var repoMetadataPath string
```

Comment:
nit: Could you move these into main to avoid global state?

Reply:
Can I keep these as they are flag value containers? `versbose` is used in many other functions, it would be annoying to pass around a local variable from main.